src/main/java/de/friday/sonarqube/gosu/plugin/rules/smells/UnnecessaryImportRule.java

```diff
@@ -17,22 +17,29 @@
 package de.friday.sonarqube.gosu.plugin.rules.smells;
 
 import de.friday.sonarqube.gosu.antlr.GosuParser;
-import de.friday.sonarqube.gosu.plugin.rules.BaseGosuRule;
 import de.friday.sonarqube.gosu.plugin.issues.GosuIssue;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Objects;
-import java.util.Set;
-import java.util.stream.Collectors;
+import de.friday.sonarqube.gosu.plugin.rules.BaseGosuRule;
 import org.antlr.v4.runtime.ParserRuleContext;
 import org.antlr.v4.runtime.tree.ParseTree;
 import org.sonar.check.Rule;
 
+import java.util.*;
+import java.util.stream.Collectors;
+
 @Rule(key = UnnecessaryImportRule.KEY)
 public class UnnecessaryImportRule extends BaseGosuRule {
 
     static final String KEY = "UnnecessaryImportRule";
 
+    private static final Set<String> GW_OBJECTS_IMPORTS_INCLUDED = new HashSet<>(Arrays.asList(
+            "typekey.",
+            "entity."
+    ));
+
+    private static final Set<String> GW_OBJECTS_IMPORTS_EXCLUDED = new HashSet<>(Collections.singletonList(
+            "entity.windowed."
```

Comment:
Instead of hard-coding this here, this should be a rule configuration that can be extended/configured on the Sonar UI. E.g. [LinesOfCodeRule](https://github.com/FRI-DAY/sonar-gosu-plugin/blob/1568c9209ad944e86743fdb067e99e091e702f43/src/main/java/de/friday/sonarqube/gosu/plugin/rules/metrics/LinesOfCodeRule.java#L32)

Reply:
reimpl